client/components/boards/boardHeader.js

```diff
@@ -15,6 +15,19 @@ Template.boardMenuPopup.events({
   }),
 });
 
+Template.boardMenuPopup.helpers({
+  exportUrl() {
+    const boardId = Session.get('currentBoard');
+    const userId = Meteor.userId();
+    const loginToken = Accounts._storedLoginToken();
+    return Meteor.absoluteUrl(`api/b/${boardId}/${userId}/${loginToken}`);
```

Comment:
Since you explicitly designed the export as the beginning point of a REST-like API, which is a good idea, I think it is worth considering the following point about the URI scheme:

* I don’t see the point of having to transmit both the `loginToken` and the `userId`, the former should be enough. This is how HTTP authentication usually work (either with a cookie on a web page, or with an explicit GET parameter on APIs) and I feel we should stick with this convention
* If we look at the [Trello API](https://developers.trello.com/advanced-reference/board) only the “mandatory method parameters” are defined in the first part of URI, the rest is defined using GET parameters. This also assume that we have a method name.

With these elements the URL would be:

```js
`api/b/${boardId}/export?tokent=${loginToken}`
```

We might also want to follow Trello and use the explicit `boards` namespace instead of its `b` shortcut:

```js
`api/boards/${boardId}/export?tokent=${loginToken}`
```

I wonder if we should also explicit the return format in the URI using something like:

```js
`api/boards/${boardId}/export.json?tokent=${loginToken}`
```

I guess that would be useful if someone wants to implement some other serialization format at some point (XML lol). But this information could also fit elsewhere like in a GET parameter (`&format=json`) and since Trello doesn’t use it we may as well follow them to facilitate plugin interpolation between Trello and Wekan.

What do you think Xavier?